MapboxNavigation/Style.swift

```diff
@@ -61,127 +24,144 @@ public class Style: NSObject {
+        apply()
     }
     
     ///  General styling
     
     /**
-     `tintColor` is used for guidance arrow, highlighted text, progress bar and
-     more.
+     Sets the tint color for guidance arrow, highlighted text, progress bar
+     and more.
      */
     public var tintColor: UIColor?
     
     /**
-     `primaryTextColor` sets the color for titles and other prominent information.
+     Sets the font family for all labels.
      */
-    public var primaryTextColor: UIColor?
-    
-    /**
-     `secondaryTextColor` sets the color for subtitles and other subtle
-     information.
-    */
-    public var secondaryTextColor: UIColor?
+    public var fontFamily: String?
     
     /**
-     `buttonTextColor` sets the text color on buttons for normal state.
+     Sets the text color on buttons for normal state.
      */
     public var buttonTextColor: UIColor?
     
     /**
-     `lineColor` sets the color of dividers and separators.
+     Sets the color of dividers and separators.
      */
     public var lineColor: UIColor?
     
     /// Maneuver view (Page view)
     
     /**
-     `maneuverViewBackgroundColor` sets the background color of the maneuver
-     view, positioned at the top.
+     Sets the background color of the maneuver view, positioned at the top.
      */
     public var maneuverViewBackgroundColor: UIColor?
     
     /**
-     `maneuverViewHeight` sets the height of the maneuver view.
+     Sets the height of the maneuver view.
      */
     public var maneuverViewHeight: CGFloat?
     
+    /**
+     Sets the font on the distance label.
+     */
+    public var distanceLabelFont: UIFont?
+    
+    /**
+     Sets the text color on the distance label.
+     */
+    public var distanceLabelTextColor: UIColor?
+    
+    /**
+     Sets the font on the destination label.
+     */
+    public var destinationLabelFont: UIFont?
+    
+    /**
+     Set the text color on the destination label.
+     */
+    public var destinationLabelTextColor: UIColor?
+    
     /// Table view (Drawer)
     
     /**
-     `headerBackgroundColor` sets the color of the drawer header, positioned at
-     the bottom.
+     Sets the color of the drawer header, positioned at the bottom.
      */
     public var headerBackgroundColor: UIColor?
     
     /**
-     `cellTitleLabelFont` sets the font of the title labels in table views.
+     Sets the font on the time remaining label.
+     */
+    public var timeRemainingLabelFont: UIFont?
+    
+    /**
+     Sets the text color on the time remaining label.
+     */
+    public var timeRemainingLabelTextColor: UIColor?
+    
+    /**
+     Sets the font on the distance remaining label.
+     */
+    public var distanceRemainingLabelFont: UIFont?
+    
+    /**
+     Sets the text color on the distance remaining label.
+     */
+    public var distanceRemainingLabelTextColor: UIColor?
+    
+    /**
+     Sets the font on the ETA label.
+     */
+    public var etaLabelFont: UIFont?
```

Comment:
Nit: We’ve tended to say `arrivalDate` or `estimatedArrivalTime` instead of `eta`.